test/integration/cart_integration_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class CartIntegrationTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test 'an unauthorized user can add an item to the cart' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.